packages/core/aqua-to-js/src/generate/interfaces.ts

```diff
@@ -134,9 +132,7 @@ export class TSTypeGenerator implements TypeGenerator {
     ];
 
     const registerServiceArgs =
-      // This wrong type comes from aqua team. We need to discuss fix with them
-      // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
-      (srvDef.defaultServiceId as DefaultServiceId).s_Some__f_value != null
+      srvDef.defaultServiceId != null
```

Comment:
I think it's tiny bit simpler if negation is used as rarely as possible (except for maybe some early returns or errors)
```suggestion
      srvDef.defaultServiceId == null ? functionOverloadsWithoutDefaultServiceId : functionOverloadsWithDefaultServiceId
```